_spinnaker_install_admin_guides/orca-sql.md

```diff
@@ -77,7 +77,7 @@ The above configuration will point Orca to your database.
 You have the option to run a dual repository with the following in `profiles/orca-local.yml`:
 
 ```yaml
-executionHistory:
+executionRepository:
```

Comment:
Can you move to kebab case?

Reply:
Of course, I just doubt that if the properties have been used with camel case, would it be good to change them to kebab case.